JPS_Ontology/ontology/ontosensorloggermobileapp/OntoSensorLoggerMobileApp.csv

```diff
@@ -0,0 +1,25 @@
+hasYComponent,Object Property,,,AccelerationVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the linear acceleration with respect to the device's cooordinate system in the y-direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasYComponent
+hasZComponent,Object Property,,,AccelerationVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the linear acceleration with respect to the device's cooordinate system in the z-direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasZComponent
+hasXComponent,Object Property,,,GravityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the acceleration vector due to gravity with respect to device's coordinate reference system in the x-direction (Short edge).,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasXComponent
+hasYComponent,Object Property,,,GravityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the acceleration vector due to gravity with respect to device's coordinate reference system in the y-direction (Long edge).,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasYComponent
+hasZComponent,Object Property,,,GravityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,only,A relationship that defines the acceleration vector due to gravity with respect to device's coordinate reference system in the z-direction (Through the phone).,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasZComponent
+hasXComponent,Object Property,,,MagneticFluxDensityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,only,A relationship that defines the direction of magnetic north with respect to the device in the x direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasXComponent
+hasYComponent,Object Property,,,MagneticFluxDensityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,only,A relationship that defines the direction of magnetic north with respect to the device in the y direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasYComponent
+hasZComponent,Object Property,,,MagneticFluxDensityVector,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,only,A relationship that defines the direction of magnetic north with respect to the device in the z direction.,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,hasZComponent
+https://www.theworldavatar.com/kg/ontodevice/measures,Object Property,,IS-A,https://www.theworldavatar.com/kg/ontodevice/Magnetometer,MagneticFluxDensityVector,,,https://www.theworldavatar.com/kg/ontosensorloggermobileapp/,measures
```

Comment:
You're specifying `IS-A`, i.e. requesting this to be a subproperty of something, but are not providing a superproperty (empty third column). Please either provide the superproperty that `measures` is meant to be a subproperty of, or remove the `IS-A`. This applies to the subsequent two definitions as well.